modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -14,18 +14,29 @@ settings and run `openshift-install` to install a cluster and `oc` to work with
 for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques]
 for a presentation on Tang.
 
-. Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
-If you miss this step, the second boot will fail.
-For example, to configure DHCP networking, identify `ip=dhcp`
-or set static networking when you add parameters to the kernel command line.
+. To configure networking when you do the RHCOS installations for your cluster:
 
+.. Add the `rd.neednet=1` kernel argument as shown in the `MachineConfig` examples below.
+
+.. Use the `ip=dhcp` kernel argument for DHCP networking.
+
+.. Add the necessary `ip=` kernel arguments for static networking.
+
+
+[IMPORTANT]
```

Comment:
I think this admonition should explicitly state that the `rd.needed=1` kernel argument is required and that other kernel arguments might be necessary depending on your network configuration.

Reply:
I struggled with this one. The statement was there before `rd.neednet=1` was added. So my intention is to say the entire step should not be skipped. But I didn't want to put it in front of the bullets because it would interrupt the intro sentence.

I like your solution and implemented your wording.